src/test/external-git/secret.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: private-git-server
+  namespace: git-server
+type: Opaque
+data:
+  username: Z2l0LXVzZXI=
+  password: c3VwZXJTZWN1cmVQYXNzd29yZA==
```

Comment:
Some scanners will 🤮 on this username/password.  Clearly this is just a test entry, but could we leave come comment stating that in case someone's scanner trips up on it?

Reply:
good point, will do 👍